### PR TITLE
Add delete torrent and data button to web ui

### PR DESCRIPTION
**Pull request by S199pWa1k9r**
Pluase add "delete torrent and data" button.

### CODE REVIEW

File: libtransmission/net.c [hunk 1]
```diff
@@ -494,7 +494,7 @@ static tr_socket_t tr_netBindTCPImpl(tr_address const* addr, tr_port port, bool
 
 #endif
 
-    if (listen(fd, 128) == -1)
+    if (listen(fd, 256) == -1)
```
Comment (ckerr) [inline libtransmission/net.c]: It's not clear to me what this is for and the commit message `modified: libtransmission/net.c` doesn't help.

I understand what the `backlog` argument is for but not why we need to double it. What's the motivation for this change?

Comment (S199pWa1k9r) [reply libtransmission/net.c]: I constantly had more than 128 incoming connections

Comment (ckerr) [reply libtransmission/net.c]: That's unrelated to a delete button. Would you be OK with moving that into a separate PR?

Comment (S199pWa1k9r) [reply libtransmission/net.c]: Yes
